src/pages/_app.tsx

```diff
@@ -13,6 +14,16 @@ export default function App({ Component, pageProps }) {
   return (
     <ErrorBoundary FallbackComponent={Error}>
       <DeviceSizeProvider>
+        <AlertBanner type="highlight">
+          <p>
+            You are viewing an internal preview and work in progress version of
+            this site.{' '}
+            <a href="https://airtable.com/shrU3eYHIOXO60o23">
```

Comment:
Total nit but should this maybe open a new window?

Reply:
I'm gonna vote yes! That way if someone has very specific feedback on what they're looking at they don't lose the page they were on.